### PR TITLE
Migrate to standalone build

### PR DESCRIPTION
**Pull request by ThisaruGuruge**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -27,6 +27,9 @@ ext.kafkaVersion = project.kafkaVersion
 ext.awaitalityVersion = "3.1.6"
 ext.avroVersion = "1.9.2"
 ext.puppycrawlCheckstyleVersion = "8.18"
+ext.stdlibFilepathVersion = project.stdlibFilepathVersion
+ext.stdlibIoVersion = project.stdlibIoVersion
+ext.stdlibSystemVersion = project.stdlibSystemVersion
```
Comment (aashikam) [inline build.gradle:33]: The build passed without the other standard library versions here? 🤔

Comment (aashikam) [reply build.gradle:33]: @BuddhiWathsala

Comment (ThisaruGuruge) [reply build.gradle:39]: Yes. Just noticed this. It passed somehow.

Comment (aashikam) [reply build.gradle:39]: Yeah. that's what!

Comment (BuddhiWathsala) [reply build.gradle:39]: I also wonder, how it got resolved dependencies at the first phase.

Comment (ThisaruGuruge) [reply build.gradle:39]: Maybe from Ballerina-lang? Anyways, I added those dependencies back.

**Review by chamil321 — APPROVED** [review]

**Pull request merged**